pkg/cpio/fs_unix.go

```diff
@@ -187,7 +187,17 @@ func inode(i Info) (Info, bool) {
 }
 
 func GetRecord(path string) (Record, error) {
-	fi, err := os.Lstat(path)
+	return GetResolvedRecord(path, false)
+}
+
+func GetResolvedRecord(path string, resolveSymlinks bool) (Record, error) {
```

Comment:
Sorry, I'm slow this morning, why do we need to resolve symlinks when in many cases that's the wrong thing to do ... what's the use case here?